### ISSUE TITLE
Can't compile on alpine linux

### ISSUE THREAD

**Issue by revington** [armon/bloomd#40]
Alpine linux uses musl. 
```
$ scons
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
gcc -o src/libbloom/bitmap.o -c -std=c99 -Wall -Werror -Wextra -O2 -D_GNU_SOURCE src/libbloom/bitmap.c
In file included from src/libbloom/bitmap.c:9:0:
/usr/include/sys/errno.h:1:2: error: #warning redirecting incorrect #include <sys/errno.h> to <errno.h> [-Werror=cpp]
 #warning redirecting incorrect #include <sys/errno.h> to <errno.h>
  ^
cc1: all warnings being treated as errors
scons: *** [src/libbloom/bitmap.o] Error 1
scons: building terminated because of errors.
```
Content of `/usr/include/sys/errno.h`
```
#warning redirecting incorrect #include <sys/errno.h> to <errno.h>
#include <errno.h>
```

**Comment by armon** [comment]
Should be fixed by 4659b3a!

**Comment by revington** [comment]
Thank you very much!
